pkg/systemd/reporting.jsx

```diff
@@ -413,20 +414,20 @@ function WorkflowRow(props) {
     let button = null;
     if (props.problemState === ProblemState.REPORTING) {
         button = (
-            <button key={"cancel_" + props.label}
-                    className="pf-c-button pf-m-danger"
+            <Button key={"cancel_" + props.label}
```

Comment:
Nothing related to this PR - but cancel buttons should not be danger I think.

Reply:
Good point, sent https://github.com/cockpit-project/cockpit/pull/13684